doc/0000-ccl_note/contributions.tex

```diff
@@ -27,7 +27,7 @@
 Javier Sanchez: Modified setup.py to allow pip installation and uninstall. \\
 Sukhdeep Singh: Contributed to the correlation functions code. \\
 An\v{z}e Slosar: Wrote and reviewed code. \\
-Tilman Tr\"oster: Wrote code for user-changable precision parameters, added distance and growth factor tests, found and fixed bugs. \\
+Tilman Tr\""oster: Wrote code for user-changable precision parameters, added distance and growth factor tests, found and fixed bugs. \\
```

Comment:
Same as above.

Reply:
Similarly to the above, I can't get what I think is the correct thing in authors.csv to produce the right thing in authors.tex, contribution.tex, or the pdf when I make the note. @elisachisari would you mind trying it on your computer?

Reply:
I would simply replace authors.csv and authors.tex with the master version. They shouldn't have changed in this PR.